docs/guide/features.md

```diff
@@ -102,6 +102,21 @@ This will provide the following type shims:
 - Types for the Vite-injected [env variables](./env-and-mode#env-variables) on `import.meta.env`
 - Types for the [HMR API](./api-hmr) on `import.meta.hot`
 
+::: tip
+To override the default typing, declare it before the triple-slash reference.
+For example, to make a default import of `*.svg` a React component, it will be like:
```

Comment:
```suggestion
To override the default typing, declare it before the triple-slash reference. For example, to make a default import of `*.svg` a React component:
```

Reply:
Thanks. My fingers are too much used to it.😅